README.rst

```diff
@@ -69,7 +69,7 @@ If you are using multiple Travis factors,
 then you can use those factors to decide what will run.
 For example, see the following ``.travis.yml`` and ``tox.ini``:
 
-.. code-block::
+.. code-block:: yaml
```

Comment:
OK, so here's the deal with that. I'm using the `README.rst` as the `long_description` for PyPI, and PyPI won't render the page _at all_ if it doesn't recognize the language. And it doesn't seem to recognize YAML.

Actually, looking back at what I originally had, it looks like I was using `yml`, so it might be fine with `yaml`. If you have some evidence that it'll work, that would be great. Otherwise, I'll just have to try it before I merge this.